bin/unix/install-dependencies/opensuse.sh

```diff
@@ -0,0 +1,39 @@
+#/bin/bash
+#
+## 
+## Usage: build-opensuse
+## 
+## This script automates the building of ownCloud documentation on the openSUSE platform.
+## While new, it should handle installing all of the required platform depencies, and afterwards
+## build the documentation. 
+##
+## Author: Matthew Setter <msetter@owncloud.com>
+##
+
+function install_dependencies()
+{
+    # first check if the repository's already been added
+    zypper lr | grep devel_languages_python &>/dev/null
+
+    if [ "$?" -eq 1 ]; then
+        echo "Adding the devel:languages:python repository to the available repository list"
+        sudo zypper addrepo --refresh http://download.opensuse.org/repositories/devel:languages:python/openSUSE_Leap_42.1/devel:languages:python.repo;
+    fi;
+
+    echo "Installing required platform dependencies..."
+    sudo zypper in -yf python-Sphinx python-rst2pdf python-sphinxcontrib-phpdomain texlive-pdfjam texlive-threeparttable texlive-wrapfig texlive-multirow
+
+    echo "Dependencies installed."
+}
+
+function build_documentation() 
+{
+    echo "Building the documentation..."
+
+    cd user_manual && make latexpdf && cd -
```

Comment:
Again, I would suggest `set -e` at the beginning of this script and then having the flexibility to run each command on its own line. Additionally my personal preference is to use `pushd` and `popd` rather than `cd ; cd -` but that's pretty subjective.